scenario/utils.ts

```diff
@@ -254,3 +254,8 @@ export async function fetchLogs(
     return contract.queryFilter(filter, fromBlock, toBlock);
   }
 }
+
+export function extractCalldata(data: string): string {
+  // Remove the first 4 bytes (function selector) of the transaction data
+  return '0x' + data.slice(10);
+}
```

Comment:
https://github.com/compound-finance/comet/blob/main/src/deploy/index.ts#L74-L77

Reply:
Oh cool, was wondering where it went. Will use this one instead

Reply:
Done